client/src/cmdhfmf.c

```diff
@@ -4250,8 +4260,8 @@ static int CmdHF14AMfCSetUID(const char *Cmd) {
 
     res = memcmp(uid, verify_uid, uidlen);
 
-    PrintAndLogEx(SUCCESS, "Old UID... %s", sprint_hex(old_uid, uidlen));
-    PrintAndLogEx(SUCCESS, "New UID... %s ( %s )",
+    PrintAndLogEx(SUCCESS, "Old UID:  %s", sprint_hex(old_uid, uidlen));
```

Comment:
No,  don't fiddle with `...`     There is my long going project of moving all":" in the client to dots instead.